### ISSUE TITLE
Add name and description to SSM Maintenance Window Tasks

### ISSUE THREAD

**Issue by nbrys** [terraform-providers/terraform-provider-aws#5702]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

Add the name and description fields in terraform when creating SSM Maintenance window tasks

The console allows to add a name and description to tasks. Terraform doesn't support these 2 fields.

* aws_ssm_maintenance_window_task

### Potential Terraform Configuration

```hcl
resource "aws_ssm_maintenance_window_task" "maintenance_task" {
  name                  = ""
  description           = ""
}
```

### References

* #0000

**Comment by bflad** [comment]
Support for these two arguments has been merged in and will be released with version 1.44.0 of the AWS provider, likely tomorrow. 👍

**Comment by bflad** [comment]
This has been released in [version 1.44.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1440-november-14-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
